api/main.go

```diff
@@ -44,6 +48,7 @@ func main() {
 	r.Handle("/cart/{workflowID}/{runID}/add", http.HandlerFunc(AddToCartHandler)).Methods("PUT")
 	r.Handle("/cart/{workflowID}/{runID}/remove", http.HandlerFunc(RemoveFromCartHandler)).Methods("PUT")
```

Comment:
From a pure HTTP perspective this operation is probably a DELETE

Reply:
I disagree with this. A DELETE is supposed to delete a resource, but in this case we don't have a distinct resource for an entry in the cart's `Items` array. There's no separate id for an entry, and technically `RemoveFromCartHandler` doesn't have to remove an entire entry.